src/core/Folder.java

```diff
@@ -0,0 +1,33 @@
+package core;
+
+import java.util.List;
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ */
+public class Folder extends Item {
+
+    public Folder (File file) {
+        super(file);
+    }
+
+    /**
+     * List of files and folders that are on a path
+     */
+    public List<String> listFilesAndFolders(String directoryName)
```

Comment:
The parameter directoryName, is only the name of the directory, is a relative path, or an absolute path?